data_validation/query_builder/random_row_builder.py

```diff
@@ -28,6 +28,7 @@
 from typing import List
 from data_validation import clients
 from io import StringIO
+from data_validation.query_builder.query_builder import QueryBuilder
```

Comment:
Nit can you order these public first then local

Reply:
Done